.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
       - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
   pull_request:
     branches:
       - master
```

Comment:
We can optionally remove this as well, since status checks will already be run by bors before merging. 

We can keep it if we still want to run all of them automatically on every PR, even though they will no longer be merge blocking.

Reply:
The payload of what bors tests and the one that CI runs are different. If I open a PR "foo" on top of master1 in July, and it gets approved in August, by which time master has advanced to master2.

CI tests master1 + foo
Bors tests master2 + foo

Both are useful to signal me (the PR author) if something is problematic, at different stages.